armi/reactor/converters/uniformMesh.py

```diff
@@ -293,82 +292,47 @@ def _filterMesh(
                 return sorted(meshList)
             meshList.pop(removeIndex)
 
-    def _getFilteredFuelTopAndBottom(self):
+    def _getFilteredMeshTopAndBottom(self, flags, bottoms=None, tops=None):
         """
         Get the bottom and top boundaries of fuel assemblies and filter them based on the ``minimumMeshSize``.
 
+        Parameters
+        ----------
+        flags : armi.reactor.flags.Flags
```

Comment:
Not super important since this is a private method, but could be helpful to specify if these input flags will be used exactly or greedily.